### ISSUE TITLE
Alexa Air Quality Monitor

### ISSUE THREAD

**Issue by sdaltons1** [custom-components/alexa_media_player#1956]
**NOTE: For a feature to be created it has to be doable in the Amazon app because we are basically replicating the app for HA. If you can't find it in the app, we probably can't create it here.**

**Is your feature request related to a problem? Please describe.**
Unable to pass humidity reading to HomeKit

**Describe the solution you'd like**
I know AirThings can pass humidity and air quality (the radon reading), along with temperature, through to HomeKit. Would like to do the same with this device. 

**Describe alternatives you've considered**
N/A

**Additional context**
I don't know how to make the coding work but I'm happy to provide logs or help in any other way.

**Comment by albal** [comment]
I am also unable to see my Air Quality Monitors in Home Assistant.  I have installed Alexa Media Player through HACS and have cloud enabled.  I am able to see my other Alexa devices (7 in total).  I have set the "Include devices connected via Echo" to enabled.  I've tried various combinations of reconfiguring the extension and rebooting and still only the 7 devices show, not the AQMs.  The AQMs are visible with data in the Alexa app.  What am I missing?
